### DISCUSSION TITLE
Project Dream Texture greyed out + trackback error

### DISCUSSION THREAD

**Discussion by platinumpies** [carson-katri/dream-textures#777] [Q&A] [answered]
Hello im having some issues setting up the latest version ( Blender 4.01 and 4.1 running dream texture v 0.3.1 )

I installed manually by unzipping into the addons folder in roaming/blender foundation/...
also installed stable-diffusion-2-1-base and stable-diffusion-2-depth

However I am not able to click the "Project Dream Texture" button, no matter what i type or how i select the mesh. 
The only thing that activates the button is unchecking the addon from preferences and re-checking it on again. This activates the button however it then passes this error: 

 Python: Traceback (most recent call last):
  File "C:\Users\willi\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\dream_textures\engine\annotations\depth.py", line 32, in _execute
    shader = gpu.shader.from_builtin('3D_UNIFORM_COLOR')
ValueError: expected a string in ('FLAT_COLOR', 'IMAGE', 'IMAGE_COLOR', 'SMOOTH_COLOR', 'UNIFORM_COLOR', 'POLYLINE_FLAT_COLOR', 'POLYLINE_SMOOTH_COLOR', 'POLYLINE_UNIFORM_COLOR'), got '3D_UNIFORM_COLOR'

Any help would be much appreciated!

**Comment by carson-katri** [comment, marked as answer]
Blender 4 is not supported by any released version of Dream Textures yet. You'll need to run off of the `main` branch if you want to use Blender 4.x.
